src/content/docs/index.mdx

```diff
@@ -39,6 +39,6 @@ import { Card, CardGrid } from "@astrojs/starlight/components";
     Put here some GIF or text.
   </Card>
   <Card title="Learn more" icon="open-book">
-    Read more in [the Alumnium Docs](https://github.com/alumnium-hq/alumnium).
+    Read more in [the Alumnium Docs](docs/).
```

Comment:
Now it makes sense to revert the changes I did and have 2 links at the top. Instead of lines 9-13, could you try these?
```
  actions:
    - text: Get started
      link: /docs
      icon: right-arrow
    - text: View on GitHub
      link: https://github.com/alumnium-hq/alumnium
      icon: external
      variant: minimal
```

Reply:
much better, thanks!
<img width="644" alt="Screenshot 2024-11-09 at 13 50 29" src="https://github.com/user-attachments/assets/5dc85c06-821f-4b42-b346-172e535d7150">